### ISSUE TITLE
🐛 [Bug Report] - Enchant Weapon - Deathfrost doesn't work properly

### ISSUE THREAD

**Issue by YifengL** [Yatzii93/TBC5MAN-Bug-Tracker#77]
# Decription: #
Deathfrost doesn't work properly on raid bosses

# How it works: # Currently Deathfrost does damage to raid bosses but doesn't reduce their attack speed.

# How it should work: # Deathfrost should reduce raid bosses' melee and ranged attack speed(does not stack with warrior TC)

# Source - You need to provide a source as evidence: #
wowwiki and wow head:
https://wowwiki.fandom.com/wiki/Enchant_Weapon_-_Deathfrost
https://www.wowhead.com/item=35498/formula-enchant-weapon-deathfrost

# Additional Information: #

**Comment by gadw** [comment]
Hopefully here is some more proof that the debuff should affect bosses in TBC.

**A)** The level cap was introduced later on in WotLK:
Wrath-logo-Small **Patch 3.0.3** (04-Nov-2008): **Effect was changed, including a level cap**.
            _Old - Permanently enchant a weapon so your damaging spells and melee hits occasionally inflict an additional 150 Frost damage and reduce the target's melee, ranged, and casting speed by 15% for 8 sec. Requires a level 60 or higher item._
Bc icon **Patch 2.4.3** (2008-07-15): Now procs off dots including consecrate.
Bc icon **Patch 2.4.0** (25-Mar-2008): Added.
(https://wowwiki.fandom.com/wiki/Enchant_Weapon_-_Deathfrost)


**B)** according to other TBC databases the levelcap was never mentioned before:
- https://tbc-twinhead.twinstar.cz/?item=35498
- https://tbcdb.com/?item=35498
- http://tbc.cavernoftime.com/item=35498


**C)** some comments from wowhead clearly states that the debuff was affecting some bosses:

By _thunderchylde_ (312 –  5·4)  on 2008/06/23 (P**atch 2.4.1**)
I can also confirm that it (currently) does work on some bosses, or at least Magtheridon is vulnerable to it. 

By _Silverado_ (4,190 –  22·42)  on 2008/06/24 (**Patch 2.4.1**)
EDIT: All of the bosses in Hyjal and Black Temple are affected by this rather despicable enchant :P

By _70855_ (594 –  5)  on 2009/01/20 (**Patch 3.0.3**)
"Slow effect does not work on targets 73 or higher."
Lame Blizzard, really lame.
I wouldn't have minded so much if you had created any decent tanking weapon enchants at 80.

(https://www.wowhead.com/spell=46578/deathfrost#comments)


**D)** some comments from Elitistjerks clearly states that the debuff was affecting bosses:

- Sapphidia (06/21/08, 5:27 PM)  comment n. 1
"(...)**Apparently it works fine on raidbosses**, so an alternative for small groups without a warrior, but its viability as a Warrior Tank enchant is very much dependant upon whether its slowing effect stacks in any way with thunderclap."

- Charsi (06/21/08, 8:58 PM)  comment n. 10
"(...)Edit: **Deathfrost can be applied to Kil'jaeden**. Whether it's worth applying is another story :P
Last edited by Charsi : 06/21/08 at 9:18 PM. "

- SSWarder (06/22/08, 12:32 AM)  comment n. 22
"I enchanted deathfrost on my old torch, and tried it out in a bt run, i found that:
-The proc does refresh itself
-**The debuff was applied to every bt boss** (...)"

- Austin (06/24/08, 3:30 PM)  comment n. 83
**I was playing around on Illidian last night and had the debuff up**, I'll have to research the logs to see if it did anything.
Here is the log, (...):
18:24'50.000Illidan Stormrage's Swing hits Mythos for 4043 Physical damage (372 blocked)
18:24'51.578Illidan Stormrage's Swing dodged by Mythos
1.578 second attack speed
18:24'51.750**Illidan Stormrage is afflicted by Deathfrost** <--Deathfrost
18:24'53.093Illidan Stormrage's Shear blocked by Mythos (...)"

- Lucke (06/25/08, 10:43 PM)  comment n. 98
"**The debuff stuck on Muru**, will post in a few if I can get it to stick on Entropius."

- Lucke (06/26/08, 1:35 AM)  comment n. 100
"**Stuck on Entropius as well."

(http://web.archive.org/web/20081110040647/http://elitistjerks.com/f15/t27347-deathfrost_its_mechanics/)

**Comment by gadw** [comment]
Hopefully here is some more proof that the debuff should affect bosses in TBC.

**A)** The level cap was introduced later on in WotLK:
Wrath-logo-Small **Patch 3.0.3** (04-Nov-2008): **Effect was changed, including a level cap**.
            _Old - Permanently enchant a weapon so your damaging spells and melee hits occasionally inflict an additional 150 Frost damage and reduce the target's melee, ranged, and casting speed by 15% for 8 sec. Requires a level 60 or higher item._
Bc icon **Patch 2.4.3** (2008-07-15): Now procs off dots including consecrate.
Bc icon **Patch 2.4.0** (25-Mar-2008): Added.
(https://wowwiki.fandom.com/wiki/Enchant_Weapon_-_Deathfrost)


**B)** according to other TBC databases the levelcap was never mentioned before:
- https://tbc-twinhead.twinstar.cz/?item=35498
- https://tbcdb.com/?item=35498
- http://tbc.cavernoftime.com/item=35498


**C)** some comments from wowhead clearly states that the debuff was affecting some bosses:

By _thunderchylde_ (312 –  5·4)  on 2008/06/23 (P**atch 2.4.1**)
I can also confirm that it (currently) does work on some bosses, or at least Magtheridon is vulnerable to it. 

By _Silverado_ (4,190 –  22·42)  on 2008/06/24 (**Patch 2.4.1**)
EDIT: All of the bosses in Hyjal and Black Temple are affected by this rather despicable enchant :P

By _70855_ (594 –  5)  on 2009/01/20 (**Patch 3.0.3**)
"Slow effect does not work on targets 73 or higher."
Lame Blizzard, really lame.
I wouldn't have minded so much if you had created any decent tanking weapon enchants at 80.

(https://www.wowhead.com/spell=46578/deathfrost#comments)


**D)** some comments from Elitistjerks clearly states that the debuff was affecting bosses:

- Sapphidia (06/21/08, 5:27 PM)  comment n. 1
"(...)**Apparently it works fine on raidbosses**, so an alternative for small groups without a warrior, but its viability as a Warrior Tank enchant is very much dependant upon whether its slowing effect stacks in any way with thunderclap."

- Charsi (06/21/08, 8:58 PM)  comment n. 10
"(...)Edit: **Deathfrost can be applied to Kil'jaeden**. Whether it's worth applying is another story :P
Last edited by Charsi : 06/21/08 at 9:18 PM. "

- SSWarder (06/22/08, 12:32 AM)  comment n. 22
"I enchanted deathfrost on my old torch, and tried it out in a bt run, i found that:
-The proc does refresh itself
-**The debuff was applied to every bt boss** (...)"

- Austin (06/24/08, 3:30 PM)  comment n. 83
**I was playing around on Illidian last night and had the debuff up**, I'll have to research the logs to see if it did anything.
Here is the log, (...):
18:24'50.000Illidan Stormrage's Swing hits Mythos for 4043 Physical damage (372 blocked)
18:24'51.578Illidan Stormrage's Swing dodged by Mythos
1.578 second attack speed
18:24'51.750**Illidan Stormrage is afflicted by Deathfrost** <--Deathfrost
18:24'53.093Illidan Stormrage's Shear blocked by Mythos (...)"

- Lucke (06/25/08, 10:43 PM)  comment n. 98
"**The debuff stuck on Muru**, will post in a few if I can get it to stick on Entropius."

- Lucke (06/26/08, 1:35 AM)  comment n. 100
"**Stuck on Entropius as well**."

(http://web.archive.org/web/20081110040647/http://elitistjerks.com/f15/t27347-deathfrost_its_mechanics/)

**Comment by Yatzii93** [comment]
![image](https://user-images.githubusercontent.com/47720837/83179858-7dcc8b00-a11a-11ea-962d-25f7bd257c15.png)
